Helper.h

```diff
@@ -1,171 +1,205 @@
-            {
-                grid.UpdateCell(testBitmap, i, 0, span);
-                ++span;
-            }
-
-            ResetScreen(grid);
-
-            bool alternate = true;
-            for (size_t i = 0; i < rows; i++)
-            {
-                for (size_t j = 0; j < cols; j++)
-                {
-                    if (alternate)
-                        grid.UpdateCell(testBitmap, i, j, 1);
-                    alternate = !alternate;
-                }
-            }
-
-            ResetScreen(grid);
-
-            alternate = false;
-            for (size_t i = 0; i < rows; i++)
-            {
-                for (size_t j = 0; j < cols; j++)
-                {
-                    if (alternate)
-                        grid.UpdateCell(testBitmap, i, j, 1);
-                }
-                alternate = !alternate;
-            }
-
-            ResetScreen(grid);
-
-            alternate = false;
-            for (size_t i = 0; i < cols; i++)
-            {
-                for (size_t j = 0; j < rows; j++)
-                {
-                    if (alternate)
-                        grid.UpdateCell(testBitmap, i, j, 1);
-                    alternate = !alternate;
-                }
-                alternate = !alternate;
-            }
-
-            ResetScreen(grid);
-
-            grid.UpdateAllCells(testBitmap);
-            ResetScreen(grid);
-
-            while (!quit_requested())
-            {
-                grid.DrawGrid();
-                process_events();
-                clear_screen();
-                refresh_screen(60);
-            }
-        }
-};
+namespace fs = std::experimental::filesystem;
+
+// To test txt configs, change to "config"
+#define CONFIG_DIR "config"
+
+/**
+ * @brief Helper class
+ * This class has been added to house orphan functions 
+ * 
+ */
+class Helper{
+    public:
+        /**
+         * @brief Convert a string to a char array
+         * 
+         * @param str String to convert to char
+         * @return char* 
+         */
+        char* string_to_char(string str)
+        {
+            int n = str.length();
+            char* name[n+1];
+            strcpy(*name, str.c_str());
+            return *name;
+        }
+
+        /**
+         * @brief Find the game folder name to store in config class.
+         * 
+         * @param entryPath 
+         * @return * string 
+         */
+        string get_folder_name(string entryPath)
+        {
+            unsigned lastPath = entryPath.find("\\", 14);
+            string dir = entryPath.substr(0, lastPath);
+            unsigned findBackSlash = dir.find("\\");
+            dir.replace(findBackSlash, 1, "/");
+
+            return dir;
+        }
+
+        /**
+         * @brief Find the config files in the games directory.
+         * 
+         * @param dir The games directory
+         * @return * vector<string> 
+         */
+        vector<string> get_config_files(string dir)
+        {
```

Comment:
Still some changes that have been removed it seems? Haven't had a look over all the changes but this function has been affected 😢